tests/istio.mk

```diff
@@ -217,3 +217,34 @@ junit-report: out_dir ${ISTIO_BIN}/go-junit-report
 # This assume istio runs in istio-system namespace, and 'skip-cleanup' was used in tests.
 dumpsys: out_dir
 	tools/dump_kubernetes.sh --output-directory ${OUT_DIR}/logs --error-if-nasty-logs
+
+# Run once for each cluster, to install helm on the cluster
+helm/init: ${HELM}
+	kubectl apply -n kube-system -f install/kubernetes/helm/helm-service-account.yaml
+	helm init --upgrade --service-account tiller
+
+# Install istio for the first time
+helm/install:
+	${HELM} install \
+	  install/kubernetes/helm/istio \
+	  --name istio-system --namespace istio-system \
+	  --set global.hub=${HUB} \
+	  --set global.tag=${TAG} \
+	  --set global.imagePullPolicy=Always \
+	  ${HELM_ARGS}
+
+# Upgrade istio. Options must be set:
+#  "make helm/upgrade HELM_ARGS="--values myoverride.yaml"
+helm/upgrade:
+	${HELM} upgrade \
+	  --set global.hub=${HUB} \
+	  --set global.tag=${TAG} \
+	  --set global.imagePullPolicy=Always \
+	  ${HELM_ARGS} \
+	  istio-system install/kubernetes/helm/istio
+
+# Delete istio installed with helm
+# Note that for Helm 2.10, the CRDs are not cleared
+helm/delete:
+	${HELM} delete --purge istio-system
+	kubectl delete -f install/kubernetes/helm/istio/templates/crds.yaml
```

Comment:
We need update of istio.io docs also after this is merged.

Reply:
Yes, I suspect there is at least one extra step ( deleting the webhook ) to get the clean state.
Will keep testing - but separate PR for extra cleanup.